src/styles/photodetails.scss

```diff
@@ -64,3 +64,10 @@ label {
   height: 64px;
   display: block;
 }
+
+.button.tag {
+  /* Small only */
+  @media screen and (max-width: 39.9375em) {
```

Comment:
You should instead add the `margin-vertical-normal` to the tag button [here](https://github.com/creativecommons/cccatalog-frontend/blob/develop/src/components/PhotoTags.vue#L6]

Reply:
Yup done :)